app/assets/stylesheets/application.css.scss

```diff
@@ -34,6 +35,216 @@ h1 {
   color: #ffffff;
 }
 
-h3 {
+h2 {
   color: #ffffff;
 }
+
+
+/* STAT SLIDER;
+ */
+.well {
+  width: 350px;
+  height: 200px;
+  margin: auto;
+}
+
+.nine-columns li {
+	width:calc(100% / 8);
+}
+
+
+/* Functional styling;
+ * These styles are required for noUiSlider to function.
+ * You don't need to change these rules to apply your design.
+ */
+.no-ui-slider {
+	width: 100%;
+}
+
+.no-ui-slider .slider-labels {
+	list-style: none;
+	padding:0;
+   margin-top: 5px;
+	margin-left: -3px;
+	width: 100%;
+	text-align: left;
+	font-size: 13px;
+}
+
+.no-ui-slider .slider-labels li {
+	color: #d3d3d3;
+	display: inline;
+	float:left;
+	cursor: pointer;
+}
+
+.no-ui-slider .slider-labels li:nth-last-child(1) {
+	position: absolute;
+	margin-left: -3px;
+	width:25px;
+}
+
+.no-ui-slider .slider-labels li:hover {
+   color: #333333;
+}
+
+.no-ui-slider .slider-labels li.active {
+   color: #333333;
+}
+
+.noUi-target,
+.noUi-target * {
+-webkit-touch-callout: none;
+-webkit-user-select: none;
+-ms-touch-action: none;
+-ms-user-select: none;
+-moz-user-select: none;
+-moz-box-sizing: border-box;
+	box-sizing: border-box;
+}
+.noUi-base {
+	width: 100%;
+	height: 100%;
+	position: relative;
+}
+.noUi-origin {
+	position: absolute;
+	right: 0;
+	top: 0;
+	left: 0;
+	bottom: 0;
+}
+.noUi-handle {
+	position: relative;
+	z-index: 1;
+  cursor: move;
+}
+.noUi-stacking .noUi-handle {
```

Comment:
Selector `noUi-stacking` should be written in lowercase with hyphens<br>Selector `noUi-handle` should be written in lowercase with hyphens